### DISCUSSION TITLE
How do you escape a dollar sign with KaTeX?

### DISCUSSION THREAD

**Discussion by talglobus** [CaiJimmy/hugo-theme-stack#978] [Q&A]
I'd expect the following text to render like the image below with `math: true`:

    The [market rate](https://www.investing.com/commodities/gold)
    for gold futures fluctuates between \$1,800 and \$2,200.

<img width="527" alt="Screenshot 2024-03-14 at 1 02 51 PM" src="https://github.com/CaiJimmy/hugo-theme-stack/assets/21287629/9a547ad0-dc22-477e-8ef1-d4f735085d2f">

Instead, it renders like this:

<img width="519" alt="Screenshot 2024-03-14 at 1 05 04 PM" src="https://github.com/CaiJimmy/hugo-theme-stack/assets/21287629/d3863028-0896-4f9a-b14e-4aafaf69abbd">

Doubling up the backslashes renders like this:

<img width="536" alt="Screenshot 2024-03-14 at 1 06 02 PM" src="https://github.com/CaiJimmy/hugo-theme-stack/assets/21287629/31c63899-32f0-49fa-9f4c-d86b01f68336">

This seems like such an easy problem, and yet the docs both here and on [katex.org](https://katex.org/) don't seem to address how to escape a dollar sign.

**Comment by CaiJimmy** [comment]
> I'd expect the following text to render like the image below with math: true:

I don't quite understand the problem. Do you want to render only the dollar sign with LaTeX?

**Reply by talglobus** [reply]
I'm trying to include dollar signs in an article with KaTeX enabled in a non-KaTeX section, and all the ways I've thought of to try to escape the dollar sign don't seem to work. See the examples of my escaping attempts above.

**Reply by chiragbharadwaj** [reply]
Having the same issue still, 1 year later. Did you ever manage to figure it out, `talglobus`?
